tests/e2e/specs/AssociationFlows.spec.js

```diff
@@ -1,24 +1,20 @@
 import {
+  connectNodesWithFlow,
   dragFromSourceToDest,
   getElementAtPosition,
   getLinksConnectedToElement,
-  connectNodesWithFlow,
 } from '../support/utils';
 
 import { direction } from '../../../src/components/nodes/association/associationConfig';
 import { nodeTypes } from '../support/constants';
 
 describe('Association Flows', () => {
-  beforeEach(() => {
-    cy.loadModeler();
-  });
-
   it('Change direction of association to none, one and both', () => {
     const directionSelectSelector = '[name=associationDirection]';
     const testDirection = {
-      none:`${ direction.none }`,
-      one: `${ direction.one }`,
-      both:`${ direction.both }`,
+      none: `${direction.none}`,
+      one: `${direction.one}`,
+      both: `${direction.both}`,
```

Comment:
Since you're touching this part of the code, it seems these don't need to be wrapped in template strings. It can be:

```js
const testDirection = {
  none: direction.none,
  one: direction.one,
  both: direction.both,
};
```

But after doing that, it's easy to see that `testDirection` is exactly the same as `direction`; so `testDirection` can be removed and `direction` can be used directly in the assertions.